shell/store/index.js

```diff
@@ -746,6 +746,11 @@ export const actions = {
       const isRancherInHarvester = (rancherManagerSupport?.value || rancherManagerSupport?.default) === 'true';
 
       commit('isRancherInHarvester', isRancherInHarvester);
+      // standalone harvester cluster needs to subscribe rancher socket to get latest token data update
```

Comment:
Not sure if there is better place to do `rancher/subscribe` or do single token subscription.